firewood/src/merkle/node.rs

```diff
@@ -457,6 +492,15 @@ impl Storable for Node {
         #[allow(clippy::unwrap_used)]
         cur.write_all(&[attrs.bits()]).unwrap();
 
+        cur.write_all(&encoded_len.to_le_bytes())?;
+        if let Some(encoded) = encoded {
+            cur.write_all(encoded)?;
+            let remaining_len = ENCODED_MAX_LEN - encoded_len as usize;
+            cur.write_all(&vec![0; remaining_len])?;
```

Comment:
If we have the encoded-length, why are we making this a constant size?

Reply:
Because the limitation of shale, this size has to be known ahead.

Reply:
Oh, otherwise you have to write a new node when encoding?